src/Driver/PgSQL/Connection.php

```diff
@@ -58,7 +60,7 @@ public function prepare(string $sql): Statement
         $result = @pg_get_result($this->connection);
         assert($result !== false);
 
-        if ((bool) pg_result_error($result)) {
+        if (pg_result_error($result) !== false) {
```

Comment:
You should revert this, trust me. 😉

Reply:
Yes, it returns an empty string if there is no error)